src/libs/ReportUtils.ts

```diff
@@ -6426,7 +6427,7 @@ function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>,
     }
     return Localize.translateLocal(translationKey, {
         formattedAmount,
-        comment: transactionDetails?.comment ?? '',
+        comment: TransactionUtils.getDescription(transaction) ?? '',
```

Comment:
NAB, should this be called `TransactionUtils.getComment`?

Reply:
I guess description works for now since we are calling it like this in App